content/docs/command-reference/diff.md

```diff
@@ -33,6 +33,14 @@ sections below for more details.
 `dvc diff` does not have an effect when the repository is not tracked by Git,
 for example when `dvc init` was used with the `--no-scm` option.
 
+By default, when comparing the current workspace to a Git commit, `dvc diff`
+includes a `not in cache` status for DVC-tracked files and directories which are
```

Comment:
I cannot reproduce this.

```console
λ echo data > data
λ dvc add data
...
λ dvc diff
Added:
    data

files summary: 1 added, 0 deleted, 0 modified

λ rm data
λ dvc diff  # No output
```

Reply:
(DVC 1.6.1 on Windows)

Reply:
> most likely need to be `dvc pull`ed

Prob no need for this info in the `diff` ref.

Reply:
PR was just merged recently, and won't show up until https://github.com/iterative/dvc/releases/tag/untagged-82c123df4a6da19799cf is released

Reply:
Yeah I saw that PR after commenting here. Unfortunately there's some problems with the development version on Windows though so I can't check this :/ could you post a brief console example?

Reply:
https://asciinema.org/a/thHUinCl2BD3rAMcU5cEYZufO